js/PulsarUIComponent.js

```diff
@@ -52,62 +119,15 @@ PulsarUIComponent.prototype.initTables = function () {
 
 PulsarUIComponent.prototype.initDataTables = function () {
     var component = this,
-        datatables = component.$html.find('.datatable:not(.table--horizontal)'),
-        datatablesHorizontal = component.$html.find('.datatable.table--horizontal');
-
+        datatables = component.$html.find('.datatable:not([data-init="false"]):not(.table--horizontal)'),
+        datatablesHorizontal = component.$html.find('.datatable.table--horizontal:not([data-init="false"])');
+        
     datatables.each(function () {
-        var $this = $(this),
-            pageLength = 25;
-
-        var select = {
-            className: 'dt-row-selected',
-            style:     'multi',
-            selector:  'td.table-selection'
-        }
-
-        var dom = '<"dataTables_top"Birf><"dataTables_actions"T>t<"dataTables_bottom"lp>';
-
-        if (!$this.data('empty-table')) {
-            $this.data('empty-table', 'There are currently no items to display');
-        }
-
-        if ($this.data('page-length')) {
-            pageLength = $this.data('page-length');
-        }
+        var $this = $(this);
 
-        if ($this.data('select') === false) {
-            dom = '<"dataTables_top"irf><"dataTables_actions"T><"dt-disable-selection"t><"dataTables_bottom"p>';
-            select = false;
-        }
+        const datatableOptions = component.getDatatableOptions($this);
```

Comment:
A quick example if that options method were to become static, this line would become 
```
const datatableOptions = PulsarUIComponent.getDatatableOptions($this);
```

Reply:
Done.